eng/pipelines/runtime.yml

```diff
@@ -75,31 +76,81 @@ jobs:
     - OSX_x64
     - Linux_x64
     - Linux_arm
+    - Linux_arm64
+    - Linux_musl_x64
     - Windows_NT_x86
     - Windows_NT_x64
     - Windows_NT_arm
     - Windows_NT_arm64
     jobParameters:
       testGroup: innerloop
+      condition: eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true)
 
 #
-# Build libraries using live CoreLib from CoreCLR
+# Build CoreCLR debug
+# Only when it contains coreclr changes
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
+    buildConfig: debug
+    platforms:
+    - Windows_NT_x64
```

Comment:
Implication here is that we only run the CoreCLR debug tests on Win64?

Reply:
Right. Note that I haven’t expanded the full matrix yet, but yeah, this is mirroring what we have on pr.yml.